src/components/Editor/SearchBar.js

```diff
@@ -473,9 +473,13 @@ const SearchBar = React.createClass({
 
   // Renderers
   buildSummaryMsg() {
-    if (this.state.symbolSearchEnabled) {
+    if (
+      this.state.symbolSearchEnabled &&
+      this.state.symbolSearchResults.length > 0
```

Comment:
So for 
- 1 result - `1 of 1 result` or `1 result`
- 1+ results - `%d of %d results`
?
@wldcordeiro  @jasonLaster